load.go

```diff
@@ -56,6 +97,20 @@ var (
 	typeScanner             = reflect.TypeOf((*sql.Scanner)(nil)).Elem()
 )
 
+func findPtrNullString(column []string, value reflect.Value) ([]interface{}, error) {
```

Comment:
I would name this function generateColumnArray, and make a comment that it supports only nullstring for now